client/src/components/App/Feedback.js

```diff
@@ -0,0 +1,9 @@
+import { PureComponent } from 'react';
+
+class Feedback extends PureComponent {
+    constructor() {
+        window.location = 'https://forms.gle/k81f2aNdpdQYeKK8A';
```

Comment:
It's recommended to use `.replace()` instead, since that also updates the browser history. More info here: https://stackoverflow.com/questions/846954/change-url-and-redirect-using-jquery/847130#847130

```suggestion
        window.location.replace('https://forms.gle/k81f2aNdpdQYeKK8A');
```